Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -218,21 +219,83 @@ public string ElementTypeEnumDescription
             }
         }
 
+        List<String> mOptionalValues = new List<string>();
+        public List<String> OptionalValues
+        {
+            get
+            {
+                return mOptionalValues;
+            }
+            set
+            {
+                mOptionalValues = value;
+            }
+        }
+
+        ObservableList<OptionalValue> mOptionalVals = new ObservableList<OptionalValue>();
         [IsSerializedForLocalRepository]
```

Comment:
1- please change "OptionalVals" name to "OptionalValuesObjectsList"
2- i didn't get the isDefualt setting in rows 243-247, why not to simplify it and set all isDefualt as false and then only for first item to set it as True?

Reply:
ok, Done